templates/sections/event-link.html.twig

```diff
@@ -0,0 +1,11 @@
+{% set right_class = 'flex-6-of-12' %}
+{% set left_class = 'flex-6-of-12' %}
+
+<div{{ attributes.addClass('centered-container', 'su-events-link', 'flex-container', settings.extra_classes) }}>
+    {% if content.main|render %}
+      <div {{ region_attributes.main.addClass('body-region', left_class) }}>
+        {{ content.main }}
+      </div>
+    {% endif %}
+   </div>
+</div>
```

Comment:
@jenbreese Is this template needed? Does the one column work?

Reply:
@sherakama this is the issue I wanted to talk to you today about. We probably don't need this template.